CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8)
 
 project(FlatBuffers)
 
+include(GNUInstallDirs)
```

Comment:
This was previously only executed if `FLATBUFFERS_INSTALL` was on. Will this work on e.g. Windows / Visual Studio?

Reply:
The module is not related with OS. It is used to avoid hard-corded path name (e.g. include, lib, bin).
Also, the CMake installation already works well on Windows.